From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sun, 3 Jan 2016 16:56:08 +0000
Subject: Makefile-silence-xargs

Silence xargs when hothasktags not installed.

This patch has been accepted for the next upstream release.

(cherry picked from commit a23c8b42b3e901da05e5de26ba9a2a4207d839f2)
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8b71387..733e718 100644
--- a/Makefile
+++ b/Makefile
@@ -37,7 +37,7 @@ clean:
 # duplicate tags with Propellor.Property. removed from the start, as we
 # often import qualified by just the module base name.
 tags:
-	find . | grep -v /.git/ | grep -v /tmp/ | grep -v /dist/ | grep -v /doc/ | egrep '\.hs$$' | xargs hothasktags | perl -ne 'print; s/Propellor\.Property\.//; print' | sort > tags  2>/dev/null || true
+	find . | grep -v /.git/ | grep -v /tmp/ | grep -v /dist/ | grep -v /doc/ | egrep '\.hs$$' | xargs hothasktags 2>/dev/null | perl -ne 'print; s/Propellor\.Property\.//; print' | sort > tags || true
 
 dist/setup-config: propellor.cabal
 	@if [ "$(CABAL)" = ./Setup ]; then ghc --make Setup; fi
